### PR TITLE
Refactored user and order to change how order is handled. Also added …

### PR DESCRIPTION
**Pull request by ryanmogauro**
I changed order.java to handle most of the checkout process. Now, user.checkout() takes in an order object, calls its checkout method, and adds it to the orders list. I also had to create some helper methods, getCart and getSubscriptionObject, for creating order objects in Bookazon.java.

### CODE REVIEW

File: src/Order.java [hunk 1]
```diff
@@ -41,6 +41,14 @@ public void setUserName(String name) {
         this.userName = name;
     }
 
+    public void checkout(String userName, Address shippingAddress, Address billingAddress){
+        this.setShippingAddress(shippingAddress);
+        this.setBillingAddress(billingAddress);
+        this.setOrderStatus("Order Placed");
+        this.setDateCreated("2024-01-01");
```
Comment (ehburdick) [inline src/Order.java]: Should this be dataCreated?

**Review by ehburdick — CHANGES_REQUESTED** [review]
Everything looked good, except, I feel as if setDateCreated should be dataCreated, may be out of the scope of the issue though

**Comment by ryanmogauro** [comment]
Just changed dateCreated and dateShipped to type LocalDate (previously String). The checkout method now sets dateCreated to LocalDate.now(), instead of the previously hard-coded date.

**Review by ehburdick — APPROVED** [review]
Looks good, glad Date is not a magic value anymore

**Pull request merged**